frontend/app/route-helpers/apply-child-route-helpers.server.ts

```diff
@@ -2,6 +2,8 @@ import { Session, redirect } from '@remix-run/node';
 import { Params } from '@remix-run/react';
 
 import { ApplyState, loadApplyState, saveApplyState } from '~/route-helpers/apply-route-helpers.server';
+import { DisabilityTaxCreditState } from '~/routes/$lang+/_public+/apply+/$id+/adult-child/disability-tax-credit';
+import { LivingIndependentlyState } from '~/routes/$lang+/_public+/apply+/$id+/adult-child/living-independently';
```

Comment:
It's either it doesn't belong here or the routes are missing.

```suggestion
import { DisabilityTaxCreditState } from '~/routes/$lang+/_public+/apply+/$id+/child/disability-tax-credit';
import { LivingIndependentlyState } from '~/routes/$lang+/_public+/apply+/$id+/child/living-independently';
```

Reply:
DTC is for adults, thus should be in `/adult` or `/adult-child`, same with living independently